apps/pocket-id/metadata/description.md

```
@@ -0,0 +1,7 @@
+Pocket ID is a simple OIDC provider that allows users to authenticate with their passkeys to your services.
+
+<img src="https://github.com/user-attachments/assets/96ac549d-b897-404a-8811-f42b16ea58e2" width="1200"/>
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)